src/components/Display.js

```diff
@@ -1,16 +1,18 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-function Display({ result }) {
-  return <span>{result}</span>;
+function Display({ next, total }) {
+  return <span>{`${next}     ${total}`}</span>;
```

Comment:
Your Display component should accept the result of the calculation (from your milestone 2) and 
result prop should be a String with default value = `0` when your app run am seeing `null`  `null` instead of `0`